scripts/postinstall.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+const commitlintrcPath = `${process.env.INIT_CWD}/.commitlintrc.js`;

```

Comment:
@linusklingzell Actually, one more thing. Since we are talking compatibility here (and specially with Windows) we should opt to `require('path')` and use this construct instead of hard coding the path separator:
```suggestion
const commitlintrcPath = path.join(process.env.INIT_CWD, '.commitlintrc.js');
```

Reply:
yes good point! It worked on both enviroment (ubuntu and windows) with or without `path.join` but better to be safe then sorry